sass/_app.scss

```
@@ -0,0 +1,73 @@
+@use 'variable' as var;
+@use 'mixin';
+
+.header{
+  max-width: var.$maxWidth;
+  height: 100px;
+  background-color: var.$navy;
+  position: relative;
+
+  & > ul {
+    display: flex;
+    gap: 10px;
+    width: 160px;
+    background-color: var.$white;
+    padding: 10px;
+    box-sizing: border-box;
+    position: absolute;
+    top: 20px;
+    right: 20px;
+
+    @each $className in var.$classNames {
+      $index_key : index(var.$classNames, $className );
+      $color : nth(var.$buttonColor, $index_key);
+      
+      & > .#{$className} {        // &(ul) > .(ulの中のliのclass名)
+          @include mixin.box($color, 40px, 40px);
+      }
+    }
+  }
+}
+
+main{
```

@Shohei-Tsu07 
```suggestion
main {
```

細かいですが半角スペースあけるように統一しましょう！（他の箇所も同様）

---

@technicalplace 
承知いたしました！